dataland-frontend/tests/e2e/utils/GeneralApiUtils.ts

```diff
@@ -155,7 +155,7 @@ export function uploadCompanyViaApiAndEuTaxonomyDataViaForm<T>(
  * @param submissionDataIntercept function that asserts content of an intercepted request
  */
 export function submitFilledInEuTaxonomyForm(
-  submissionDataIntercept: (request: CyHttpMessages.IncomingHttpRequest) => void
+  submissionDataIntercept?: (request: CyHttpMessages.IncomingHttpRequest) => void
```

Comment:
I've checked the code:  `submitFilledInEuTaxonomyForm` is used twice in our code, and `submissionDataIntercept` is always passed when it is used.
So you can revert this change here.

Or is there anothe reason for this change?

Reply:
👍